dts/arm/st/f7/stm32f7.dtsi

```diff
@@ -639,6 +639,13 @@
 			status = "disabled";
 			label = "DMA_2";
 		};
+
+		fmc: stm32-fmc@60000000 {
```

Comment:
@0xA0000000 ?

Reply:
Yes, that looks correct, I misunderstood the values in the data sheet.